test/classes/constructors/dflt-ctor-generic-uninit-field-var-err-1.good

```diff
@@ -1,2 +0,0 @@
-dflt-ctor-generic-uninit-field-var-err-1.chpl:5: error: unresolved call 'A.init()'
-dflt-ctor-generic-uninit-field-var-err-1.chpl:1: note: candidates are: A.init(v)
```

Comment:
All of the tests up until here (in this commit) have a corresponding initializer test that references them.  It would be good to remove the references since these tests will no longer exist.